sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -0,0 +1,146 @@
+# Azure Cognitive Services Health Insights Radiology Insights client library for Python
+[Health Insights][health_insights] is an Azure Applied AI Service built with the Azure Cognitive Services Framework, that leverages multiple Cognitive Services, Healthcare API services and other Azure resources.
+
+[Radiology Insights][radiology_insights_docs] is a model that aims to provide quality checks as feedback on errors and inconsistencies (mismatches) and ensures critical findings are identified and communicated using the full context of the report. Follow-up recommendations and clinical findings with measurements (sizes) documented by the radiologist are also identified.
+
+## Getting started
+
+### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Cognitive Services Health Insights instance.
+
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
+
+### Installing the module
+
+```bash
+python -m pip install azure-healthinsights-radiologyinsights
+```
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+| SDK version | Supported API version of service |
+|-------------|----------------------------------|
+| 1.0.0b1     | 2023-09-01-preview               |
+
+
+### Authenticate the client
+
+#### Get the endpoint
+
+You can find the endpoint for your Health Insights service resource using the [Azure Portal][azure_portal] or [Azure CLI][azure_cli]
+
+
+```bash
+# Get the endpoint for the Health Insights service resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+#### Get the API Key
+
+You can get the **API Key** from the Health Insights service resource in the Azure Portal.
+Alternatively, you can use **Azure CLI** snippet below to get the API key of your resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create a RadiologyInsightsClient with an API Key Credential
+
+Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter to authenticate the client:
+
+```python 
+\<!-- SNIPPET:sample_critical_result_inference_async.create_radiology_insights_client-->
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.clinicalmatching import ClinicalMatchingClient
```

Comment:
This snippet is importing the wrong client from the wrong package

Reply:
fixed